environments/kubernetes/manifests/thanos-compactor-statefulSet.yaml

```diff
@@ -0,0 +1,45 @@
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  labels:
+    app.kubernetes.io/name: thanos-compactor
+  name: thanos-compactor
+  namespace: monitoring
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app.kubernetes.io/name: thanos-compactor
+  serviceName: thanos-compactor
+  template:
+    metadata:
+      labels:
+        app.kubernetes.io/name: thanos-compactor
+    spec:
+      containers:
+      - args:
+        - compact
+        - --wait
+        - --retention.resolution-raw=16d
+        - --retention.resolution-5m=42d
+        - --retention.resolution-1h=180d
+        - --objstore.config=$(OBJSTORE_CONFIG)
+        env:
+        - name: OBJSTORE_CONFIG
+          valueFrom:
+            secretKeyRef:
+              key: thanos.yaml
+              name: thanos-objectstorage
+        image: quay.io/thanos/thanos:v0.6.1
```

Comment:
v0.7.0?

Reply:
Yes. We've moved to v0.7.0 on upstream kube-thanos but still override the image in this repo: https://github.com/observatorium/configuration/blob/master/environments/kubernetes/kube-thanos.libsonnet#L19

I guess it's safe to move to v0.7.0 however that should be another PR, imho.

Reply:
:+1: